packages/KTable/KTable.vue

```diff
@@ -39,6 +42,38 @@
 </template>
 
 <script>
+export const defaultSorter = (key, sortKey, sortOrder, items) => {
```

Comment:
could use jsdoc here. I'm a little confused on difference between key and sortKey. is sortKey basically "the previous key used to sort"? I would imagine that sortKey is what you would be sorting on but afaict `key` is actually what is being used to sort.

Reply:
yeah, i think i will rename it `sortKey` and `previousKey` it makes more sense

Reply:
actually i know its confusing, but the param in the `defaultSorter` was renamed to `previousKey` - there's another `sortKey` thats passed in as a prop and that one is more in line with what the name says - its the current or default sort key so i kept that one the same